### PR TITLE
update 3r waterleak ts

### PR DESCRIPTION
**Pull request by 3reality-support**

### CODE REVIEW

File: src/devices/third_reality.ts [hunk 1]
```diff
@@ -91,6 +91,18 @@ const definitions: Definition[] = [
         fromZigbee: [fz.ias_water_leak_alarm_1, fz.battery],
         toZigbee: [],
         ota: ota.zigbeeOTA,
+        extend: [
```
Comment (Koenkk) [inline src/devices/third_reality.ts:94]: Could you also expose these features?

Comment (Koenkk) [reply src/devices/third_reality.ts:94]: > hi，we don't need this temporarily, it will be added later

If it's not needed, why was the cluster added?

Comment (3reality-support) [reply src/devices/third_reality.ts:94]: Sorry, I didn't explain clearly. Adding this cluster is only used on the dev console menu for the time being, and not on the expose menu.

Comment (Koenkk) [reply src/devices/third_reality.ts:94]: I see, thanks!

File: src/devices/third_reality.ts [hunk 2]
```diff
@@ -209,6 +221,17 @@ const definitions: DefinitionWithExtend[] = [
...
     },
+    {
+        zigbeeModel: ['3RSM0147Z'],
+        model: '3RSM0147Z',
+        vendor: 'Third Reality',
+        description: 'Soil sensor',
+        fromZigbee: [fz.temperature, fz.humidity],
+        toZigbee: [],
+        exposes: [e.temperature(), e.humidity()],
+        extend: [battery({voltage: true}), forcePowerSource({powerSource: 'Battery'})],
```
Comment (Koenkk) [inline src/devices/third_reality.ts:232]:
```suggestion
        extend: [battery({voltage: true}), forcePowerSource({powerSource: 'Battery'}), temperature(), humidity()],
```

And remove exposes, fromZigbee and toZigbee

**Comment by Koenkk** [comment]
Could you also submit a picture for the docs? This can be done by:
1. Create a fork by clicking [here](https://github.com/Koenkk/zigbee2mqtt.io/fork)
2. Go to the `public/images/devices` directory, *Add file* -> *Upload files*
3. Upload the files and press *Commit changes*
4. Press *Contribute* -> *Open pull request* -> update title/description -> *Create pull request*

Make sure that:
- The filename is `3RSM0147Z.png`
- The size is 512x512
- The background is transparent (use e.g. [Adobe remove background](https://new.express.adobe.com/tools/remove-background))

File: src/devices/third_reality.ts (hunk 2 above)
Comment (3reality-support) [reply src/devices/third_reality.ts]: It has been modified. Please check it again.，thanks

**Comment by Koenkk** [comment]
Thanks!

**Pull request merged**